src/components/content/content.component.js

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import PropTypes from "prop-types";
+import {
+  StyledContent,
+  StyledContentTitle,
+  StyledContentBody,
+} from "./content.style.js";
+
+const Content = ({
+  variant = "primary",
+  children,
+  title,
+  inline = false,
+  align = "left",
+  titleWidth,
+  bodyFullWidth = false,
+}) => {
```

Comment:
Minor but saying as this component has no logic there's no need for the curly brackets and return statement, i.e.
```jsx
const Content = ({
  ...
}) => (
  <StyledContent>
  ...
  </StyledContent>
);
```

Reply:
You don't have to change this but just FYI